pkg/controllers/machinesetsync/machineset_sync_controller.go

```diff
@@ -307,8 +317,11 @@ func (r *MachineSetSyncReconciler) reconcileMAPIMachineSetToCAPIMachineSet(ctx c
 	newCAPIMachineSet.SetNamespace(r.CAPINamespace)
 	newCAPIMachineSet.Spec.Template.Spec.InfrastructureRef.Namespace = r.CAPINamespace
 	newCAPIMachineSet.OwnerReferences = []metav1.OwnerReference{clusterOwnerRefence}
-	// Set the paused annotation on the new CAPI MachineSet, as we want to create it paused.
-	annotations.AddAnnotations(newCAPIMachineSet, map[string]string{capiv1beta1.PausedAnnotation: ""})
+
+	if authoritativeAPI == machinev1beta1.MachineAuthorityMachineAPI {
+		// Set the paused annotation on the new CAPI MachineSet, if the authoritativeAPI is Machine API.
```

Comment:
I might be tempted to rewrite this comment include the *why*: to call out we're explicitly handling the case where we run an initial MAPI -> CAPI sync

Reply:
Ah right, but what I meant was a comment explaining why the normal case was branched (from the POV of the principle of least surprise): when we're MachineAuthorityClusterAPI we don't want to set paused. I don't think that's immediately clear reading the code, especially with the way around we're having the normal case branching.(I think normal case branching makes sense for code simplicity)